compiler/tests/org/jetbrains/kotlin/cli/JarOutputTest.kt

```diff
@@ -80,4 +96,16 @@ class DeterministicOutputTest : TestCaseWithTmpdir() {
             entry = zis.nextEntry
         }
     }
+
+    private fun assertNoModuleInfoClass(jar: File) {
+        val zis = JarInputStream(FileInputStream(jar))
+        var entry: ZipEntry? = zis.nextEntry
+        while (entry != null) {
```

Comment:
I think we can iterate over all entries a bit easier with `for (entry in JarFile(jar).entries())`. Consider maybe improving the code of the other test functions in this file similarly as well

Reply:
Happy to refine it, I will take a look if there are more test files  can be refined like this.